samples/Forms/LibVLCSharp.Forms.MediaElement/LibVLCSharp.Forms.Sample.MediaElement.iOS/AppDelegate.cs

```diff
@@ -26,5 +30,26 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
             return base.FinishedLaunching(app, options);
         }
+
+        public override UIInterfaceOrientationMask GetSupportedInterfaceOrientations(UIApplication application, [Transient] UIWindow forWindow)
```

Comment:
I try to keep the sample code as lean as possible. This causes necessary boilerplate that will likely be missed by users

Reply:
You could probably detect this purely from Xamarin.Forms with https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/layouts/device-orientation?tabs=macos#reacting-to-changes-in-orientation

Reply:
Okay, I'll take a look.